pkg/build/controller/factory/factory.go

```diff
@@ -115,6 +115,16 @@ func (factory *BuildControllerFactory) Create() controller.RunnableController {
 	}
 }
 
+func formatError(msg string) string {
+	if len(msg) > 0 {
+		msg = strings.ToUpper(msg)[:1] + msg[1:]
+	}
+	if !strings.HasSuffix(msg, ".") {
+		msg = msg + "."
```

Comment:
this will add "." to an empty string, this if block needs to move underneath the other one